src/Logger.js

```diff
@@ -62,81 +63,109 @@ export default class Logger {
       return acc;
     }, {});
 
-    this.winston = new winston.Logger({
+    this.winston = winston.createLogger({
       transports: [
         new winston.transports.Console({
-          stderrLevels: this.stderrLevels,
-          handleExceptions: false,
-          colorize: this.colorize,
-          formatter: this.isProduction
-            ? (...args) => this.formatJSONMessage(...args)
-            : undefined,
-          level: this.level
+          stderrLevels: this.stderrLevels
         })
       ],
+      format: this.format(),
+      level: this.level,
       levels: this.winstonLevels,
-      colors: this.colors
+      handleExceptions: false,
+      exitOnError: this.exitOnError
```

Comment:
Is this the right way to do this? I found [`rejectionHandlers`](https://github.com/winstonjs/winston?tab=readme-ov-file#rejections) in the docs. Is `handleExceptions: false` the mechanism to instruct the default transport to be used for unhandled promise rejections?

Reply:
yes i think i was wrong about that.  Property `handleExceptions` is not available at this position, i needs to be in a `transports`: eg.

`new winston.transports.Console({
          stderrLevels: this.stderrLevels,
          handleExceptions: false
        })`
just like we had it before.
I though with `exitOnError` we log the stack trace before exiting?